### PR TITLE
[Derma] User view

### PR DESCRIPTION
**Pull request by overlordbleck**

### CODE REVIEW

**Comment by overlordbleck** [comment]
Jenkins doesn't like this... :jenkins_cloudy:

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this you garbage program

File: lua/cfcfactions/core/cl/dermas/minis/cl_userview.lua [hunk 1]
```diff
@@ -0,0 +1,30 @@
+local Panel = {}
+
+local function addUserInfo( listView, data )
```
Comment (ghost) [inline lua/cfcfactions/core/cl/dermas/minis/cl_userview.lua:3]: listView is a local variable inside Panel:Init()

Will have to make it a global variable to access it later (Like when user data changes serverside)

**Review by ghost — APPROVED** [review]

**Pull request merged**